src/module/actor/SR5Actor.ts

```diff
@@ -746,6 +743,11 @@ export class SR5Actor extends Actor {
                     return {...skill, id};
             }
         }
+
```

Comment:
What's the reason for moving active skills to last in lookup order?

Reply:
Most likely the "normal" call will fail because it is not an active skill. Secondly, active skills are currently the largest batch. While they are not as large as to have an impact of performance I already sorted it for estimated amount